pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -357,6 +363,14 @@ def setup(
             )
         self.fh = fh
 
+        allowed_index_types = pd.core.indexes.period.PeriodIndex
```

Comment:
This should be a tuple in order to semantically match the variable name and show that it can be expanded in the future if needed (needs to change the f-string to support an iterable too)

Reply:
'black' is removing the `tuple` unfortunately since there is only 1 element in it right now. Can we expand this later when we know what other types will be supported?

Reply:
You can force the tuple to stay there by doing `allowed_index_types = (pd.core.indexes.period.PeriodIndex,)`

Reply:
ok, fixed now.